spring-xd-analytics.tex

```diff
@@ -65,9 +65,10 @@ \section{Analytics}
 \par
 
 The Shell based processor/sink implementation can be used to run predictive
-analytics algorithms written in other languages such as Python and dynamically
+analytics algorithms written in other languages such as Python and R which dynamically
 update the model which makes predictive analytics more adaptive.
 
-\subsection{Spark MLLib}
-A spark MLLib algorithm can be run as a sparkApp job in XD.
-
+\subsection{Spark MLlib}
+A spark MLlib algorithm can be run as a sparkApp job in XD. With the streaming
```

Comment:
Having some trouble understanding the Spark MLlib section. Is it that a stream triggers the sparkApp job?